utils.ts

```diff
@@ -63,3 +64,56 @@ export function mapToChannels(channelNames: string[], guild: Guild): TextChannel
     .filter(channel => channel)
     .filter(channel => !_.includes(blacklisted, channel.name.toLowerCase())) as TextChannel[];
 }
+
+export async function initializeBotAudit(bot: CommandoClient)
+{
+  try {
```

Comment:
I would like to not have a specific unique role for the channel as discussed in chat